src/modules/45-dashboards/pages/__tests__/DashboardsCommonHeaderPage.test.tsx

```diff
@@ -71,4 +73,16 @@ describe('DashboardsHeader', () => {
 
     await waitFor(() => expect(screen.queryByText(sidebarQueryText)).toBeInTheDocument())
   })
+
+  test('it should close the getting started sidebar when clicking the getStarted X button', async () => {
+    renderComponent()
+    const getStartedButton = screen.getByText('getStarted') as HTMLButtonElement
+    userEvent.click(getStartedButton)
```

Comment:
You don't typically have to be explicit that it's a button.
```suggestion
    const getStartedButton = screen.getByText('getStarted')
    userEvent.click(getStartedButton)
```

But if it is, try using `getByRole` instead.